policyengine_us/variables/gov/states/co/ccap/entry/co_ccap_hhs_smi_eligible.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class co_ccap_hhs_smi_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Colorado Child Care Assistance Program through the household size state median income"
+    reference = (
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=19",
+        "https://docs.google.com/spreadsheets/d/1WzobLnLoxGbN_JfTuw3jUCZV5N7IA_0uvwEkIoMt3Wk/edit#gid=1350122430",
+    )
+    definition_period = MONTH
+    # defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        spm_unit = tax_unit.spm_unit
+        monthly_agi = np.round(
+            tax_unit("adjusted_gross_income", period.this_year)
+            / MONTHS_IN_YEAR,
+            2,
+        )
+        year = period.start.year
+        month = period.start.month
+        if month >= 10:
```

Comment:
can skip the line above
```suggestion
        if period.start.month >= 10:
```